l10n_it_fiscalcode/README.rst

```diff
@@ -0,0 +1,72 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+IT Fiscal Code
+==============
+
+This module extends the functionality of partner to fit italian laws and mores
+and to allow you to computation Fiscal code computation for partner
+
+Installation
+============
+
+To install this module, you need to:
+
+#. Install Python Depends codicefiscale https://pypi.python.org/pypi/codicefiscale
```

Comment:
It would be better to write the link using RST syntax.
Please see GitHub Issues in https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst#bug-tracker as example.